packages/utils/src/index.ts

```diff
@@ -1 +1,2 @@
 export * from "./decorators"
+export * as ContainerUtils from "./container"
```

Comment:
Would make sense that this is a `CommonTypes`? `common.ts`, as well as `CommonUtils`
Container is not a "domain"

Reply:
Yes indeed, I could place it in common 👍 I ll do that

Reply:
done